provider/schema/column.go

```diff
@@ -275,3 +278,38 @@ func (c Column) checkType(v interface{}) bool {
 
 	return false
 }
+
+func (c Column) GetMeta() *ColumnMeta {
```

Comment:
I would call this `Meta()` to make it [idiomatic](https://golang.org/doc/effective_go#Getters).